src/js/edition-switch.js

```diff
@@ -4,6 +4,8 @@ const defaultClassName = 'o-header__edition-switch';
 class EditionSwitcher {
 	constructor(headerEl, config = {editionswitchClassName: defaultClassName}) {
 		this.editionSwitcherEl = headerEl.querySelector(`.${config.editionswitchClassName}`);
+		if (!this.editionSwitcherEl) return;
```

Comment:
Related to Alice talking about style: do we allow conditionals without curly brackets?